### ISSUE TITLE
Amber19SB and cmaps

### ISSUE THREAD

**Issue by Valdes-Tresanco-MS** [ParmEd/ParmEd#1228]
It needs to also remove the `cmaps` check here to handle `Amber19SB` topologies
https://github.com/ParmEd/ParmEd/blob/252c5ba52ebcd57509feb525d6da9e00f9d1ffe3/parmed/amber/_amberparm.py#L289-L300

## Test code
```python
# Read a GROMACS topology file with Amber19SB ff
com_top = parmed.load_file('topol.top') 
com_amb_prm = parmed.amber.AmberParm.from_structure(com_top)
com_amb_prm
```
Happy to open a PR

Mario S.

**Comment by swails** [comment]
Is this just in the v3.4 branch, or also master?

**Comment by Valdes-Tresanco-MS** [comment]
In both, although the one I have tested is the v3.4 branch

`v3.4`
https://github.com/ParmEd/ParmEd/blob/84acb693e305654369466ddddaf10a52fb47e51b/parmed/amber/_amberparm.py#L307-L318
`master`
https://github.com/ParmEd/ParmEd/blob/5e48857403ce8fd7848abed4c58fbf677781f97b/parmed/amber/_amberparm.py#L289-L300

**Comment by swails** [comment]
Thanks for the report. Fixed by #1233
